recipes-examples/wolfssl-py/wolf-py-tests/wolf-py-tests_5.6.0.bb

```diff
@@ -0,0 +1,53 @@
+SUMMARY = "wolfSSL Python, a.k.a. wolfssl, is a Python module that encapsulates \
+           wolfSSL's SSL/TLS library."
+DESCRIPTION = "The wolfSSL SSL/TLS library is a lightweight, portable, \
+               C-language-based library targeted at IoT, embedded, and RTOS \
+               environments primarily because of its size, speed, and feature \
+               set. It works seamlessly in desktop, enterprise, and cloud \
+               environments as well."
+HOMEPAGE = "https://www.wolfssl.com/products/wolfssl"
+BUGTRACKER = "https://github.com/wolfSSL/wolfssl-py/issues"
+SECTION = "libs"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://LICENSING.rst;md5=e4abd0c56c3f6dc95a7a7eed4c77414b"
+
+
+
+
+SRC_URI = "git://github.com/wolfSSL/wolfssl-py.git;nobranch=1;protocol=https;rev=0a8a76c6d426289d9019e10d02db9a5af051fba8"
+
+
+DEPENDS += " wolfssl-py \
+             wolfcrypt-py \
+           "  
+
+S = "${WORKDIR}/git"
+
+
+WOLFSSL_PY_TEST_DIR = "${WORKDIR}/../../wolfssl-py/5.6.0-r0/git/*"
```

Comment:
Use something like we do with wolfCLU for finding the wolfssl directory or wolfssl-py test directory if possible.

--with-wolfssl=${COMPONENTS_DIR}/${PACKAGE_ARCH}/wolfssl/usr